src/test-util/mock-imported-components.js

```diff
@@ -17,6 +17,17 @@ function isComponent(value) {
   );
 }
 
+/**
+ * Is the import source local (versus third-party)?
+ *
+ * @param {string} source
+ * @returns {boolean}
+ */
+function isLocalSource(source) {
+  // Does the source start with a literal '.'?
+  return !!source.match(/^\./);
```

Comment:
This could use `source.startsWith('.')`. In that case it would be so simple you could just move it to be a variable in the calling function rather than a helper.

Reply:
Cool, done.